examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.spark.streaming.receiver.example;
```

Comment:
we need to add license header to every file. you can run `mvn license:format` to do so.

Reply:
Ok  will do 
Thanks 
Kishore

Reply:
Made changes to add License details in New files as mentioned using mvn:license:format